index.html

```diff
@@ -1429,6 +1429,27 @@ <h4>Text</h4>
           <p>The source language of <a>Translation</a> <a>Text</a> objects and, where applicable,
             <a>Original</a> <a>Text</a> objects
             is indicated using the <a>Text Language Source</a> property.</p>
+          <p>If the <a>Text</a> object, or part of it, represents something more specific, or different,
+            to its parent <a>Script Event</a>, it can have a different <a>Represents</a> property.</p>
+          <aside class="note">
+            <p>Where possible, text representing
+            different parts of the <a>related media object</a> should be put into
+            <a>Text</a> objects in different <a>Script Events</a> each with their own <a>Represents</a> property.
+            The option to assign <a>Represents</a> directly to <a>Text</a> objects or parts of <a>Text</a> objects
+            is provided in case this is impractical.</p>
+            <p>For example, consider a <a>Script Event</a> authored to describe both
+              non-textual parts of the video image and
+              visual text content, that is intended to be voiced in a single pass.
+              If this were split into two or more <a>Script Events</a> then it would be necessary
+              to predict with audio sample accuracy when the recording will switch between
+              the each <a>Script Event</a>, in order to get correct times.
+              Doing would likely lead to timing errors, though the times could be adjusted after the recording. 
```

Comment:
I know it's a note but the notion of "predicting duration with sample accuracy" or "timing errors" feels too detailed and unnecessary.
 ```suggestion
              If this were split into two or more <a>Script Events</a> then it would be necessary
              to assign an end time to the first one and a start time to the second one, 
              which may not be simple.
```